src/azul/terra.py

```diff
@@ -91,129 +93,119 @@
 log = logging.getLogger(__name__)
 
 
+class SourceType(enum.Enum):
+    bigquery = 'bigquery'
+    parquet = 'parquet'
+
+
+class SourceDomain(enum.Enum):
+    gcp = 'gcp'
```

Comment:
Can you use `enum.auto` to avoid the duplication?

Reply:
Same problem:

```
>>> TDRSourceSpec.Domain('gcp')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nadove/.pyenv/versions/3.11.9/lib/python3.11/enum.py", line 714, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nadove/.pyenv/versions/3.11.9/lib/python3.11/enum.py", line 1137, in __new__
    raise ve_exc
ValueError: 'gcp' is not a valid TDRSourceSpec.Domain
```